include/client/kb-categories.inc.php

```diff
@@ -41,7 +41,7 @@
             <li><i></i>
             <div style="margin-left:45px">
             <h4><?php echo sprintf('<a href="faq.php?cid=%d">%s %s</a>',
-                $C->getId(), Format::htmlchars($C->getFullName()),
+                $C->getId(), Format::htmlchars($C->getLocalName()),
```

Comment:
@protich Would it be better for me to keep this as `getFullName()` and change `getFullName()` to return `getLocalName()` first instead of `self::getNameById`?

Reply:
No - that will force translation when it might not be needed. `getFullName` should return what's stored in database.